### PR TITLE
Added merriweather fonts. Added body size css classes.

### PR DESCRIPTION
**Pull request by jonathandick**
Added css styles for serif font-family (merriweather) and body size

### CODE REVIEW

File: src/typography/typography.stories.html [hunk 1]
```diff
@@ -21,7 +21,20 @@ <h3>Code example: Font</h3>
     </div>
   </section>
   <section>
-    <h3>Code examples: text decoration</h3>
+    <h3>Code example: Serif</h3>
+    <p>OpenMRS uses merriweather for serif style.</p>
```
Comment (joeldenning) [inline src/typography/typography.stories.html]: Maybe modify line 9 of this file to clarify that Roboto applies by default to all text, but Merriweather is also available?

File: src/typography/typography.css [hunk 2]
```diff
@@ -64,6 +68,39 @@ body {
...
+
+.omrs-type-body-serif {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 1.0625;
+  line-height: 147.6%;
+  color: var(--omrs-color-brand-black);
+}
+
+.omrs-type-body-small {
```
Comment (joeldenning) [inline src/typography/typography.css]: Looks like we're missing a few classes?

`omrs-type-body-mini`
`omrs-type-body-subheading`

And that maybe `omrs-type-body-subheading` should have `text-transform: uppercase;`?? Based on the designs in figma. Not sure if we should force uppercase or not, but if we do want to do that it is done with text-transform. See https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform

File: src/typography/typography.css [hunk 3]
```diff
@@ -64,6 +68,39 @@ body {
...
+
+.omrs-type-body-regular {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 1.0625rem;
+  line-height: 147.6%;
+  color: var(--omrs-color-brand-black);
+}
+
+.omrs-type-body-serif {
```
Comment (joeldenning) [inline src/typography/typography.css]: I think there's a typo in the JIRA issue and that this css class should not exist. When you want serif font, you should do `<div class="omrs-type-body omrs-type-serif"></div>`. Not `<div class="omrs-type-body-serif"></div>`

File: src/typography/typography.css (hunk 2 above)
Comment (jonathandick) [reply src/typography/typography.css]: These two classes were removed by Greg (they have a line through them in the figma file)

Comment (joeldenning) [reply src/typography/typography.css]: ah okay 👍

File: src/typography/typography.stories.html (hunk 1 above)
Comment (jonathandick) [reply src/typography/typography.stories.html]: Done

File: src/typography/typography.css (hunk 3 above)
Comment (joeldenning) [reply src/typography/typography.css]: This was not addressed in your most recent commit ^^

Comment (jonathandick) [reply src/typography/typography.css]: Removed and recommitted

**Pull request merged**